index.html

```diff
@@ -31,12 +31,133 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
         </p>
         <p class="connection">LET’S CONNECT</p>
         <ul>
-            <li><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
+            <li ><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
             <li><a href="#"><img src="image/Linkedin icon.png" alt="linkedin"></a></li>
             <li><a href="#"><img src="image/Vector (11).png" alt="mail"></a></li>
             <li><a href="#"><img src="image/Vector (12).png" alt="github"></a></li>
             <li><a href="#"><img src="image/Vector (13).png" alt="peace"></a></li>
         </ul>
     </section>
+    <!-- WORK SECTION -->
+    <section class="work">
+        <!-- CARD 1 -->
+        <div class="card-1">
+            <img src="image/Snapshoot Portfolio.png" alt="image1" >
+            <h1 class="tonic">Tonic</h1>
+            <div class="canop">
+                <p class="can-info">CANOPY</p>
+                <div class="point"></div>
+                <p class="back">Back End Dev</p>
+                <div class="point"></div>
+                <p class="back">2015</p>
+            </div>
+            <div class="languages">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavaScript", "HTML", and "CSS") should be represented using a `ul` element, with each technology as a separate `li` element. Currently, you are using `div` and `p` tags for this purpose. Please update your code to use the appropriate HTML structure to represent the list of technologies.
- [ ] Additionally, the font styles for the project technologies do not match the design specifications. Please review the design carefully and update your code to ensure consistent font face, size, weight, and other relevant styles for the technologies.
![image](https://github.com/mar4444/re-portfolio/assets/55840999/30d195bf-1274-4691-8e5a-0a86b859ba4d)
- [ ] Furthermore, the current padding around the project technologies does not match the desired spacing in the design. I recommend adjusting the padding to align with the design specifications.
![image](https://github.com/mar4444/re-portfolio/assets/55840999/e7dae599-9157-44c2-a6d7-1011229ffc9d)
Please make these adjustments to enhance the visual consistency and accuracy of the project technologies section. Keep up the good work! 👏🌟